src/typings/v8-inspect-profiler.d.ts

```diff
@@ -31,7 +31,18 @@ declare module 'v8-inspect-profiler' {
 		stop(afterDelay?: number): PromiseLike<ProfileResult>;
```

Comment:
it's merged.

Reply:
thanks.

Reply:
update.

Reply:
get call back from extension.

Reply:
new comment.

Reply:
123

Reply:
345

Reply:
sdadasdasd

Reply:
now I know Kai made some comments successfully

Reply:
Indeed

Reply:
great

Reply:
*italic*

Reply:
asdf

Reply:
**bold** _italic_

Reply:
cool

Reply:
**bold** _italic_